README.md

```diff
@@ -75,6 +75,9 @@ you can even use `npx` to view this list of `awesome-npx` tools:
 ### [npm-check](https://github.com/dylang/npm-check) - interactively update npm dependencies
 `npx npm-check`
 `npx npm-check --skip-unused --update`
+### [npm-show-versions](https://github.com/iarna/npm-show-versions) - show versions, dist-tags and restrictions
+`npm npm-show-versions awesome-npx`
```

Comment:
Should these lines start with `npx` instead of `npm`?

Reply:
lol, yes